aggregator/src/main/java/com/redhat/cloud/notifications/processors/email/aggregators/DailyEmailAggregationJob.java

```diff
@@ -34,24 +37,26 @@
     @Inject
     EndpointEmailSubscriptionResources subscriptionResources;
 
-    private static final String DEFAULT_STRING_SERIALIZER = "org.apache.kafka.common.serialization.StringSerializer";
-    private static final String DEFAULT_BROKER_URL = "localhost:9092";
-    private static final String DEFAULT_ACKS = "1";
+    @ConfigProperty(name = "notifications.aggregator.serializer")
+    String serializer;
 
-    @Scheduled(identity = "dailyEmailProcessor", cron = "{email.subscription.daily.cron}")
-    public void processDailyEmail(ScheduledExecution se) {
-        // Only delete on the largest aggregate time frame. Currently daily.
-        final List<EmailAggregation> aggregatedEmails = processAggregateEmails(se.getScheduledFireTime());
+    @ConfigProperty(name = "notifications.aggregator.broker-url")
+    String brokerUrl;
 
-        Map<String, String> config = new HashMap<>();
-        config.put("bootstrap.servers", DEFAULT_BROKER_URL);
-        config.put("key.serializer", DEFAULT_STRING_SERIALIZER);
-        config.put("value.serializer", DEFAULT_STRING_SERIALIZER);
-        config.put("acks", DEFAULT_ACKS);
+    @ConfigProperty(name = "notifications.aggregator.acks")
+    String acks;
+
+    @ConfigProperty(name = "notifications.aggregator.topic.name")
+    String topicName;
 
+    @Scheduled(identity = "dailyEmailProcessor", cron = "{notifications.aggregator.email.subscription.periodic.cron}")
+    public void processDailyEmail(ScheduledExecution se) throws JsonProcessingException {
+        List<EmailAggregation> aggregatedEmails = processAggregateEmails(se.getScheduledFireTime());
+
+        Map<String, String> config = createKafkaConfig();
         KafkaProducer<String, String> producer = KafkaProducer.create(Vertx.vertx(), config);
 
-        KafkaProducerRecord<String, String> records = KafkaProducerRecord.create("mytopic", aggregatedEmails.toString());
+        KafkaProducerRecord<String, String> records = KafkaProducerRecord.create(topicName, new ObjectMapper().writeValueAsString(aggregatedEmails));
```

Comment:
The `ObjectMapper` instance provided by Quarkus should be used here (it can be injected).